selfdrive/boardd/test_boardd_loopback.py

```diff
@@ -0,0 +1,49 @@
+"""Enable the DEBUG_WITH_PANDA_LOOPBACK define in boardd.cc before test."""
```

Comment:
Can we move this to an environment variable instead of needing to rebuild? How about BOARDD_LOOPBACK?